tests/rustdoc.rs

```diff
@@ -147,7 +147,7 @@ fn rustdoc_only_bar_dependency() {
 
 
 #[test]
-fn rustdoc_same_name_err() {
+fn rustdoc_same_name_documents_lib() {
```

Comment:
Wow, I didn't realize we have separate `cargo doc` and `cargo rustdoc` commands!

Reply:
Yup! 🤷‍♀️